Libraries/Opc.Ua.Client/Subscription.cs

```diff
@@ -2145,6 +2184,21 @@ private void VerifySubscriptionState(bool created)
             }
         }
 
+        /// <summary>
+        /// Validates the sequence number of the incoming publish request.
+        /// </summary>
+        private bool ValidSequentialPublishMessage(IncomingMessage message)
+        {
+            // If sequential publishing is enabled, only release messages in perfect sequence. 
+            if (message.SequenceNumber <= m_lastSequenceNumberProcessed + 1 ||
+                // reconnect / transfer subscription case
+                m_resyncLastSequenceNumberProcessed)
+            {
+                return true;
```

Comment:
I'm not sure if I remember your code style guides, but why not return the boolean expression directly? 
```csharp
 // If sequential publishing is enabled, only release messages in perfect sequence. 
return message.SequenceNumber <= m_lastSequenceNumberProcessed + 1 ||
                // reconnect / transfer subscription case
                m_resyncLastSequenceNumberProcessed;
```

Reply:
yep, that were easier.